containers/datalab/Dockerfile.in

```diff
@@ -97,7 +97,8 @@ RUN mkdir -p /datalab/web && \
         node-uuid@1.4.3 \
         bunyan@1.4.0 \
         socket.io@1.3.6 \
-        tcp-port-used@0.1.2 && \
+        tcp-port-used@0.1.2 \
+        googleapis@2.1.3 && \
```

Comment:
Does this stuff really help over just manually crafting JSON and issuing HTTP requests. Whenever I look at this, I get the feeling these simple wrappers bring in tons of dependencies ... which have some impact on the disk size, but more importantly bloat into the node process space.